akka-stream-tests/src/test/java/akka/stream/javadsl/FlowTest.java

```diff
@@ -382,6 +383,55 @@ public void mustBeAbleToUseMerge() throws Exception {
   }
 
   @Test
+  public void mustBeAbleToUseAsyncMerge() throws Exception {
```

Comment:
Obvisouly it tests `.fromCompletionStage`, but `mustBeAbleToUseCompletionStage` doesn't help more for me.